charters/UNITY/charter.md

```diff
@@ -0,0 +1,54 @@
+# Unity Benchmarking Service
+
+## Description
+
+The Unity benchmarking service automates the process of running and benchmarking pipelines.
+
+## Objectives
+
+The Unity benchmarking service aims to automate benchmarking for HCA pipelines, enforce standards for consistent reporting for historic comparison and enable the public to participate in suggesting improvements to pipelines by empowering them to implement improvements and test results using the same infrastructure as the HCA data processing pipelines team. The Unity benchmarking service will be used by those making pipelines for the HCA data processing service and will be public, so anyone can benchmark their implemented ideas against HCA pipelines.
+
+## In-scope
+
+* Benchmarking for HCA pipelines will be automated and included in Unity
+* Benchmarking data will be available through Unity but not controllable by the users
+* Benchmarking data can change without notice but will be consistent in format and schema with previous data or notice will be given to users
+* Leaderboards showing how pipelines differ will be created as different pipelines for the same assay are collected
```

Comment:
_Differences between pipelines for the same assay will be presented in leaderboards._ ?

Reply:
Updated to "Leaderboards showing how pipelines that process the same assay differ in performance will be created as different pipelines for the same assay are collected"